static/js/vue-cdr-access/src/mixins/fullRecordUtils.js

```diff
@@ -0,0 +1,94 @@
+import breadCrumbs from '@/components/full_record/breadCrumbs.vue';
+import modalMetadata from '@/components/modalMetadata.vue';
+import thumbnail from '@/components/full_record/thumbnail.vue';
+import isEmpty from 'lodash.isempty';
+
+export default {
+    components: { breadCrumbs, modalMetadata, thumbnail },
+
+    data() {
+        return {
+            displayMetadate: false,
+            showFullAbstract: false
+        }
+    },
+
+    props: {
+        recordData: Object
+    },
+
+    computed: {
+        abstractLinkText() {
+            return this.showFullAbstract ? this.$t('full_record.read_less'): this.$t('full_record.read_more');
+        },
+
+        truncateAbstract() {
+            return this.recordData.briefObject.abstractText !== undefined &&
+                this.recordData.briefObject.abstractText.length > 350;
+        },
+
+        truncatedAbstractText() {
+            if (this.truncateAbstract && !this.showFullAbstract) {
+                return this.recordData.briefObject.abstractText.substring(0, 350);
+            }
+
+            return this.recordData.briefObject.abstractText;
+        },
+
+        isDeleted() {
+            if (this.recordData.markedForDeletion) {
+                return 'deleted';
+            }
+            return '';
+        },
+
+        childCount() {
+            if (this.recordData.briefObject.countMap === undefined) {
+                return 0;
+            }
+            return this.recordData.briefObject.countMap.child;
+        },
+
+        pluralizeItems() {
```

Comment:
Could also make this into a "displayChildCount" method that returns the count plus units as a single operation, since they always go together